src/app/src/util/util.js

```diff
@@ -68,6 +75,18 @@ export const downloadListItemCSV = (list, items) =>
 export const downloadFacilitiesCSV = (facilities, options) =>
     DownloadCSV(createFacilitiesCSV(facilities, options), 'facilities.csv');
 
+export const createFacilitiesXLSX = (facilities, options) => {
+    const data = formatDataForCSV(facilities, options);
+    const worksheet = XLSX.utils.aoa_to_sheet(data);
+    const workbook = XLSX.utils.book_new();
+    XLSX.utils.book_append_sheet(workbook, worksheet, 'Facilities');
```

Comment:
Consider the lowercase `facilities` as the sheet name, which is consistent with the name of the file `facilities.csv` and the all lowercase column headers.